packages/pipeline/src/parsers/relayer_registry/index.ts

```diff
@@ -0,0 +1,36 @@
+import * as R from 'ramda';
+
+import { RelayerResponse, RelayerResponseNetwork } from '../../data_sources/relayer-registry';
+import { Relayer } from '../../entities';
+
+/**
+ * Parses a raw relayer registry response into an array of Relayer entities.
+ * @param rawResp raw response from the relayer-registry json file.
+ */
+export function parseRelayers(rawResp: Map<string, RelayerResponse>): Relayer[] {
+    const parsedAsObject = R.mapObjIndexed(parseRelayer, rawResp);
+    return R.values(parsedAsObject);
+}
+
+function parseRelayer(relayerResp: RelayerResponse, uuid: string): Relayer {
+    const relayer = new Relayer();
+    relayer.uuid = uuid;
+    relayer.name = relayerResp.name;
+    relayer.homepageUrl = relayerResp.homepage_url;
+    relayer.appUrl = relayerResp.app_url;
+    const mainnet = getMainNetwork(relayerResp);
```

Comment:
```suggestion
    const mainNetworkRelayerInfo = getMainNetworkRelayerInfo(relayerResp);
```

Simply naming this `mainnet` is confusing.